### PR TITLE
Fix the order of results

### PR DESCRIPTION
**Pull request by lorenzoPrimi**
Fixes https://github.com/ooni/probe/issues/1162

## Proposed Changes

  - Add fixed sorting algorithm

### CODE REVIEW

File: app/src/main/java/org/openobservatory/ooniprobe/model/database/Result.java [hunk 1]
```diff
@@ -78,10 +86,30 @@ public static void deleteAll(Context c) {
...
+	 * Whatsapp - Telegram - Facebook for Instant Messaging
+	 * Ndt - Dash - HIRL - HHFM for Performance
+	 * Psiphon - Tor for Circumvention
+	 */
 	public List<Measurement> getMeasurementsSorted() {
-		if (measurements == null)
-			measurements = SQLite.select().from(Measurement.class).where(Measurement_Table.result_id.eq(id), Measurement_Table.is_rerun.eq(false), Measurement_Table.is_done.eq(true)).orderBy(Measurement_Table.is_anomaly, false).orderBy(Measurement_Table.is_failed, false).orderBy(Measurement_Table.id, true).queryList();
-		return measurements;
+		if (WebsitesSuite.NAME.equals(test_group_name))
+			return SQLite.select().from(Measurement.class).where(Measurement_Table.result_id.eq(id), Measurement_Table.is_rerun.eq(false), Measurement_Table.is_done.eq(true)).orderBy(Measurement_Table.is_anomaly, false).orderBy(Measurement_Table.is_failed, false).orderBy(Measurement_Table.id, true).queryList();
```
Comment (bassosimone) [inline app/src/main/java/org/openobservatory/ooniprobe/model/database/Result.java:98]: Line too long, please wrap! Also, please, add `{` and `}`.

File: app/src/main/java/org/openobservatory/ooniprobe/model/database/Result.java [hunk 2]
```diff
@@ -151,6 +179,21 @@ public AbstractSuite getTestSuite() {
 		}
 	}
 
+	public String[] getTestOrder() {
+		switch (test_group_name) {
+			case WebsitesSuite.NAME:
+				return new String[] {WebConnectivity.NAME};
```
Comment (bassosimone) [inline app/src/main/java/org/openobservatory/ooniprobe/model/database/Result.java:185]: Please, no space between `[]` and `{`

Comment (lorenzoPrimi) [reply app/src/main/java/org/openobservatory/ooniprobe/model/database/Result.java:185]: done

File: app/src/main/java/org/openobservatory/ooniprobe/model/database/Result.java (hunk 1 above)
Comment (lorenzoPrimi) [reply app/src/main/java/org/openobservatory/ooniprobe/model/database/Result.java:98]: done

**Pull request merged**